src/50_Samples_%26_Templates/Product.html

```diff
@@ -260,13 +260,13 @@
       <!-- ## Navigation -->
       <!-- Use the [amp-sidebar](/components/amp-sidebar/) component to give customers the chance to quickly jump to any of your product categories.-->
       <amp-sidebar id="drawermenu" layout="nodisplay">
-        <div class="topheader">
-          <a href="/" class="home">Products by Example</a>
+        <div class="topheader m1">
+          <a href="/" class="caps text-decoration-none">Products by Example</a>
         </div>
         <hr/>
-        <h4 class="category expanded"><a href="/samples_templates/product_listing/preview/">Fruit</a></h4>
-        <h4 class="category"><a href="/samples_templates/product_listing/preview/">Vegetable</a></h4>
-        <h4 class="category"><a href="/samples_templates/product_listing/preview/">More</a></h4>
+        <h4><a class="m1 caps text-decoration-none" href="/samples_templates/product_listing/preview/">Fruit</a></h4>
```

Comment:
This will make all the links in the same line.
Would it be ok to add a <br/> after each one?